schema-guide.md

```diff
@@ -23,7 +23,7 @@ title: My Research Software
 
 ### Typical example
 
-For most software however, it is relatively easy to expand the minimal case with some more information like the version, the date when it was last published, some keywords, etc.:
+For most software however, it is relatively easy to expand the minimal case with some more important information like the version, the date when it was last published, some keywords, etc.:
```

Comment:
a bit ambiguous: do you mean "more important" or "more, important" either way I don;t think this addition is needed

Reply:
Okay, I see, was trying to stress, the importance of having this metadata. Will give this another shot, and if this fails, remove.

Reply:
Done in https://github.com/citation-file-format/citation-file-format/pull/270/commits/3c3ec0c9f4ac515df99a24e5ae3154103dcbbc88.